src/JobQueue.cc

```diff
@@ -114,7 +114,7 @@ bool sendJobToMainLoop(PyObject *pyFunc) {
 
   PyGILState_Release(gstate);
   return true;
-}  
```

Comment:
can we remove this bogus change?

Reply:
This was done by autoformat. https://github.com/Distributive-Network/PythonMonkey/issues/283